module-2/app.ts

```diff
@@ -0,0 +1,21 @@
+import express from 'express';
+import * as dotenv from 'dotenv';
+import userRouter from './routes/userRouter';
+import { errorHandler, pageNotFound } from './middleware/errorHandlers';
+
+dotenv.config();
+
+const app = express();
+const PORT = process.env.PORT || 3000;
+
+app.use(express.json());
+
+app.get('/', (req, res) => {
+  res.redirect('/users');
+});
+
+app.use('/users', userRouter);
+app.use(pageNotFound);
+app.use(errorHandler);
+
+app.listen(PORT, () => process.stdout.write(`App is listening on ${PORT}`));
```

Comment:
NIT: console.log is enough here but up to you

Reply:
process.stdout.write() is so badass, that I thought I will use it here :)